<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Annotating variants with OpenCRAVAT, using Bioconductor • BiocOpenCRAVAT</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Annotating variants with OpenCRAVAT, using Bioconductor">
<meta property="og:description" content="BiocOpenCRAVAT">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-93043521-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-93043521-1');
</script>
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">BiocOpenCRAVAT</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.0.5</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/BiocOpenCRAVAT.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/devel.html">oc2bioc -- notes for developers</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/seandavi/BuildABiocWorkshop2020">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="BiocOpenCRAVAT_files/accessible-code-block-0.0.1/empty-anchor.js"></script><script src="BiocOpenCRAVAT_files/htmlwidgets-1.5.1/htmlwidgets.js"></script><script src="BiocOpenCRAVAT_files/jquery-1.12.4/jquery.min.js"></script><link href="BiocOpenCRAVAT_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="BiocOpenCRAVAT_files/datatables-binding-0.15/datatables.js"></script><link href="BiocOpenCRAVAT_files/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="BiocOpenCRAVAT_files/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="BiocOpenCRAVAT_files/dt-core-1.10.20/js/jquery.dataTables.min.js"></script><link href="BiocOpenCRAVAT_files/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet">
<script src="BiocOpenCRAVAT_files/crosstalk-1.1.0.1/js/crosstalk.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Annotating variants with OpenCRAVAT, using Bioconductor</h1>
                        <h4 class="author">Vince Carey^[<a href="mailto:stvjc@channing.harvard.edu%7D" class="email">stvjc@channing.harvard.edu}</a>
</h4>
            
      
      <small class="dont-index">Source: <a href="https://github.com/vjcitn/BiocOpenCRAVAT/blob/master/vignettes/BiocOpenCRAVAT.Rmd"><code>vignettes/BiocOpenCRAVAT.Rmd</code></a></small>
      <div class="hidden name"><code>BiocOpenCRAVAT.Rmd</code></div>

    </div>

    
    
<div id="introduction-to-biocopencravat" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction-to-biocopencravat" class="anchor"></a>Introduction to BiocOpenCRAVAT</h1>
<p>The oc2bioc package (currently at github.com/vjcitn/oc2bioc) incorporates the open-cravat python modules.</p>
<div id="displaying-and-searching-the-resources" class="section level2">
<h2 class="hasAnchor">
<a href="#displaying-and-searching-the-resources" class="anchor"></a>Displaying and searching the resources</h2>
<p>We acquire the metadata for all OpenCRAVAT ‘modules’ in real time using <code>populate_module_set</code>.</p>
<div class="sourceCode" id="cb1"><pre class="downlit">
<span class="fu"><a href="https://rdrr.io/r/base/library.html">library</a></span>(<span class="kw">oc2bioc</span>)
<span class="co">#&gt; Loading required package: reticulate</span>
<span class="co">#&gt; Loading required package: httr</span>
<span class="kw">modset</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/oc2bioc/man/populate_module_set.html">populate_module_set</a></span>()
<span class="kw">modset</span>
<span class="co">#&gt; OpenCRAVATModuleSet, created  Tue Sep  8 10:57:03 2020 </span>
<span class="co">#&gt; There are 282 modules.</span>
<span class="co">#&gt; Module types/counts:</span>
<span class="co">#&gt;      aggregator       annotator          common       converter           group </span>
<span class="co">#&gt;               1             179               2               7               6 </span>
<span class="co">#&gt;          mapper  postaggregator        reporter webviewerwidget </span>
<span class="co">#&gt;               1               3               6              77</span>
</pre></div>
<p>We transformed the module set object to a searchable HTML table: </p>
<div id="htmlwidget-cd944818ace978f97abd" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-cd944818ace978f97abd">{"x":{"filter":"none","data":[["23andme-converter","abraom","aggregator","ancestrydna-converter","arrvars","biogrid","brca1_func_assay","cadd_exome","cancer_hotspots","cardioboost","cgc","cgl","chasmplus","chasmplus_ACC","chasmplus_ACC_mski","chasmplus_BLCA","chasmplus_BLCA_mski","chasmplus_BRCA","chasmplus_BRCA_mski","chasmplus_CESC","chasmplus_CESC_mski","chasmplus_CHOL","chasmplus_CHOL_mski","chasmplus_COAD","chasmplus_COAD_mski","chasmplus_DLBC","chasmplus_DLBC_mski","chasmplus_ESCA","chasmplus_ESCA_mski","chasmplus_GBM","chasmplus_GBM_mski","chasmplus_HNSC","chasmplus_HNSC_mski","chasmplus_KICH","chasmplus_KICH_mski","chasmplus_KIRC","chasmplus_KIRC_mski","chasmplus_KIRP","chasmplus_KIRP_mski","chasmplus_LAML","chasmplus_LAML_mski","chasmplus_LGG","chasmplus_LGG_mski","chasmplus_LIHC","chasmplus_LIHC_mski","chasmplus_LUAD","chasmplus_LUAD_mski","chasmplus_LUSC","chasmplus_LUSC_mski","chasmplus_MESO","chasmplus_MESO_mski","chasmplus_OV","chasmplus_OV_mski","chasmplus_PAAD","chasmplus_PAAD_mski","chasmplus_PCPG","chasmplus_PCPG_mski","chasmplus_PRAD","chasmplus_PRAD_mski","chasmplus_READ","chasmplus_READ_mski","chasmplus_SARC","chasmplus_SARC_mski","chasmplus_SKCM","chasmplus_SKCM_mski","chasmplus_STAD","chasmplus_STAD_mski","chasmplus_TGCT","chasmplus_TGCT_mski","chasmplus_THCA","chasmplus_THCA_mski","chasmplus_THYM","chasmplus_THYM_mski","chasmplus_UCEC","chasmplus_UCEC_mski","chasmplus_UCS","chasmplus_UCS_mski","chasmplus_UVM","chasmplus_UVM_mski","chasmplus_group","chasmplus_mski","chasmplus_mski_group","civic","civic_gene","clingen","clinvar","cosmic","cosmic_gene","cravat-converter","csvreporter","cvdkp","dbsnp","dbsnp-converter","denovo","encode_tfbs","esp6500","ess_gene","exac_gene","excelreporter","fathmm","flank_seq","ftdna-converter","genehancer","gerp","geuvadis","ghis","gnomad","gnomad3","gnomad_gene","go","grasp","gtex","gwas_catalog","haploreg_afr","haploreg_amr","haploreg_asn","haploreg_eur","haploreg_group","hg19","hg19wgs","hg38","hg38wgs","hgdp","intact","interpro","javierre_promoters","linsight","litvar","loftool","metalr","metasvm","mirbase","mupit","mutation_assessor","mutpanning","mutpred1","ncbigene","ncrna","ndex","ndex_group","ndex_signor","oldcravat-converter","omim","pharmgkb","phastcons","phdsnpg","phi","phylop","polyphen2","prec","pseudogene","pubmed","repeat","revel","rvis","segway","segway_adrenal","segway_blood","segway_brain","segway_breast","segway_esc","segway_esc_derived","segway_fat","segway_gi_colon","segway_gi_duodenum","segway_gi_intestine","segway_gi_rectum","segway_gi_stomach","segway_group","segway_heart","segway_ipsc","segway_kidney","segway_liver","segway_lung","segway_muscle","segway_muscle_leg","segway_ovary","segway_placenta","segway_skin","segway_stromal_connective","segway_thymus","segway_vascular","sift","siphy","stdoutreporter","tagsampler","target","textreporter","thousandgenomes","thousandgenomes_ad_mixed_american","thousandgenomes_african","thousandgenomes_east_asian","thousandgenomes_european","thousandgenomes_group","thousandgenomes_south_asian","trinity","tsvreporter","uk10k_cohort","uniprot","varmeta","vcf-converter","vcfinfo","vcfreporter","vest","vista_enhancer","wgbase","wgbiogrid","wgbrca1_func_assay","wgcancer_hotspots","wgcgc","wgcgl","wgcglsummary","wgchasmplus","wgcircossummary","wgcivic","wgcivic_gene","wgclingen","wgclinvar","wgcodingvsnoncodingsummary","wgcosmic","wgcosmic_gene","wgdbsnp","wgdenovo","wgesp6500","wgess_gene","wgexac_gene","wgfathmm","wggenehancer","wggerp","wgghis","wggnomad","wggnomad3","wggnomad_gene","wggo","wggosummary","wggrasp","wggtex","wggwas_catalog","wghaploreg_afr","wghaploreg_amr","wghaploreg_asn","wghaploreg_eur","wghaplotypes","wghg19","wghgvs","wgigv","wgintact","wginterpro","wglinsight","wgloftool","wglollipop","wgmupit","wgmutation_assessor","wgmutpred1","wgncbigene","wgncrna","wgndex","wgndexchasmplussummary","wgndexvestsummary","wgnote","wgomim","wgpharmgkb","wgphastcons","wgphdsnpg","wgphi","wgphylop","wgprec","wgpseudogene","wgpubmed","wgrevel","wgrvis","wgsiphy","wgsosamplesummary","wgsosummary","wgtarget","wgthousandgenomes","wgtopgeneschasmplussummary","wgtopgenessummary","wgtopgenesvestsummary","wguk10k_cohort","wgvcfinfo","wgvest"],["23andMe Converter","ABRaOM","Aggregator","AncestryDNA Converter","Arrhythmia Channelopathy Variants","BioGRID","BRCA1 Saturation Genome Editing Scores","CADD Exome","Cancer Hotspots","CardioBoost","Cancer Gene Census","Cancer Gene Landscape","CHASMplus","CHASMplus ACC","CHASMplus ACC MSK-IMPACT","CHASMplus BLCA","CHASMplus BLCA MSK-IMPACT","CHASMplus BRCA","CHASMplus BRCA MSK-IMPACT","CHASMplus CESC","CHASMplus CESC MSK-IMPACT","CHASMplus CHOL","CHASMplus CHOL MSK-IMPACT","CHASMplus COAD","CHASMplus COAD MSK-IMPACT","CHASMplus DLBC","CHASMplus DLBC MSK-IMPACT","CHASMplus ESCA","CHASMplus ESCA MSK-IMPACT","CHASMplus GBM","CHASMplus GBM MSK-IMPACT","CHASMplus HNSC","CHASMplus HNSC MSK-IMPACT","CHASMplus KICH","CHASMplus KICH MSK-IMPACT","CHASMplus KIRC","CHASMplus KIRC MSK-IMPACT","CHASMplus KIRP","CHASMplus KIRP MSK-IMPACT","CHASMplus LAML","CHASMplus LAML MSK-IMPACT","CHASMplus LGG","CHASMplus LGG MSK-IMPACT","CHASMplus LIHC","CHASMplus LIHC MSK-IMPACT","CHASMplus LUAD","CHASMplus LUAD MSK-IMPACT","CHASMplus LUSC","CHASMplus LUSC MSK-IMPACT","CHASMplus MESO","CHASMplus MESO MSK-IMPACT","CHASMplus OV","CHASMplus OV MSK-IMPACT","CHASMplus PAAD","CHASMplus PAAD MSK-IMPACT","CHASMplus PCPG","CHASMplus PCPG MSK-IMPACT","CHASMplus PRAD","CHASMplus PRAD MSK-IMPACT","CHASMplus READ","CHASMplus READ MSK-IMPACT","CHASMplus SARC","CHASMplus SARC MSK-IMPACT","CHASMplus SKCM","CHASMplus SKCM MSK-IMPACT","CHASMplus STAD","CHASMplus STAD MSK-IMPACT","CHASMplus TGCT","CHASMplus TGCT MSK-IMPACT","CHASMplus THCA","CHASMplus THCA MSK-IMPACT","CHASMplus THYM","CHASMplus THYM MSK-IMPACT","CHASMplus UCEC","CHASMplus UCEC MSK-IMPACT","CHASMplus UCS","CHASMplus UCS MSK-IMPACT","CHASMplus UVM","CHASMplus UVM MSK-IMPACT","CHASMplus","CHASMplus MSK-IMPACT","CHASMplus MSK-IMPACT","CIViC","CIViC Gene","ClinGen Gene","ClinVar","COSMIC","COSMIC Gene","Cravat Converter","CSV Reporter","Cardiovascular Disease Knowledge Portal","dbSNP","dbSNP Converter","Denovo-DB","ENCODE TFBS","ESP6500","Essential Genes","ExAC Gene and CNV","Excel Reporter","FATHMM","Flanking Sequence","FamilyTreeDNA Converter","GeneHancer","GERP++","Geuvadis eQTLs","GHIS","gnomAD","gnomAD3","gnomAD Gene","Gene Ontology","GRASP","GTEx eQTLs","GWAS Catalog","HaploReg African","HaploReg American","HaploReg Asian","HaploReg European","HaploReg","hg19 coordinates","UCSC hg19 Whole Genome Sequence","UCSC hg38 Gene Mapper","UCSC hg38 Whole Genome Sequence","HGDP","IntAct","InterPro","Promoter IR","LINSIGHT","LitVar","LoFtool","MetaLR","MetaSVM","miRBase","MuPIT","Mutation Assessor","Mutpanning","MutPred","NCBI Gene","ncRNA","NDEx NCI Cancer Pathways","NDEx","NDEx SIGNOR","OldCravat Converter","OMIM","PharmGKB","Phast Cons","PhD-SNPg","p(HI)","PhyloP","PolyPhen-2","P(rec)","Pseudogene","PubMed","Repeat Sequences","REVEL","RVIS","Segway","Segway Adrenal","Segway Blood","Segway Brain","Segway Breast","Segway Esc","Segway Esc Derived","Segway Fat","Segway GI Colon","Segway GI Duodenum","Segway GI Intestine","Segway GI Rectum","Segway GI Stomach","Segway","Segway Heart","Segway Ipsc","Segway Kidney","Segway Liver","Segway Lung","Segway Muscle","Segway Muscle Leg","Segway Ovary","Segway Placenta","Segway Skin","Segway Stromal Connective","Segway Thymus","Segway Vascular","SIFT","SiPhy","STDOUT Reporter","Tag Sampler","TARGET","Text Reporter","1000 Genomes","1000 Genomes-Ad Mixed American","1000 Genomes-African","1000 Genomes-East Asian","1000 Genomes-European","1000 Genomes","1000 Genomes-South Asian","Trinity CTAT","TSV Reporter","UK10k Cohorts","UniProt","Variant Metadata","VCF Converter","VCF Info","VCF Reporter","VEST4","VISTA Enhancer Browser","Variant Annotation","BioGRID","BRCA1 Functional Assay","Cancer Hotspots","Cancer Genome Census","Cancer Gene Landscape","Cancer Gene Landscape Summary","CHASMplus","Circos Summary","CIViC","CIViC Gene","ClinGen","Clinvar","Coding vs Noncoding Summary","Cosmic","Cosmic Gene","dbSNP","denovo-db","ESP6500","Essential Gene","ExAC Gene","FATHMM","GeneHancer","GERP++","GHIS","gnomAD","gnomAD3","gnomAD Gene","Gene ontology","Gene Ontology Summary","GRASP","GTEx","GWAS Catalog","Haploreg African","Haploreg American","Haploreg Asian","Haploreg European","Haplotype Information","hg19 info widget","HGVS","IGV","IntAct","InterPro","LINSIGHT","LoFtool","Protein diagram","MuPIT","Mutation Assessor","MutPred","NCBI Gene","Non-coding RNA diagram","NDEx","Cancer Pathways Enriched for CHASMplus Drivers (NDEx)","NDEX Enriched Pathogentic Pathways","Note","OMIM","PharmGKB","Phast Cons","PhDSNPg","P(HI)","PhyloP","P(rec)","Pseudogene","PubMed","REVEL","RVIS","SiPhy","Sequence Ontology Sample Summary","Sequence Ontology Summary","Target","1000 Genomes","Top Genes CHASMplus Summary","Genes Ranked by Total Variants","Top Genes VEST Summary","UK10K Cohort","VCF info widget","VEST"],["converter","annotator","aggregator","converter","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","group","annotator","group","annotator","annotator","annotator","annotator","annotator","annotator","converter","reporter","annotator","annotator","converter","annotator","annotator","annotator","annotator","annotator","reporter","annotator","annotator","converter","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","group","annotator","common","mapper","common","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","group","annotator","converter","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","group","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","annotator","reporter","postaggregator","annotator","reporter","annotator","annotator","annotator","annotator","annotator","group","annotator","annotator","reporter","annotator","annotator","postaggregator","converter","postaggregator","reporter","annotator","annotator","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget","webviewerwidget"],["kmoad@insilico.us.com","","kmoad@insilico.us.com","kmoad@insilico.us.com","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","ctokhei1@alumni.jh.edu","support@cravat.us","support@cravat.us","clingen@clinicalgenome.org","support@cravat.us","support@cravat.us","support@cravat.us","kmoad@insilico.us.com","rkim@insilico.us.com","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","","evsserver@uw.edu","support@cravat.us","support@cravat.us","rkim@insilico.us.com","support@cravat.us","support@cravat.us","kmoad@insilico.us.com","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","","","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","gwas-info@ebi.ac.uk.","luke@lukeward.net","luke@lukeward.net","luke@lukeward.net","luke@lukeward.net","luke@lukeward.net","rkim@insilico.us.com","support@cravat.su","support@cravat.su","support@cravat.su","","support@cravat.us","support@cravat.us",null,"asiepel@cshl.edu","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","mirbase@manchester.ac.uk","","support@cravat.us","support@cravat.us","vpejaver@uw.edu","","","","rkim@insilico.us.com","","kmoad@insilico.us.com","support@cravat.us","feedback@pharmgkb.org","support@cravat.us","emidio.capriotti@unibo.it","support@cravat.us","support@cravat.us","iadzhubey@rics.bwh.harvard.edu","support@cravat.us","","pubmedcentral@ncbi.nlm.nih.gov","","joseph.rothstein@mssm.edu","support@cravat.us","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","michael.hoffman@utoronto.ca","ngpc4@gis.a-star.edu.sg","support@cravat.us","rkim@insilico.us.com","rkim@insilico.us.com","support@cravat.us","rkim@insilico.us.com","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","support@cravat.us","rkim@insilico.us.com","support@cravat.us","support@cravat.us","support@cravat.us","kmoad@insilico.us.com","kmoad@insilico.us.com","rkim@insilico.us.com","","vtdelapuente@lbl.gov","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","kmoad@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","kmoad@insilico.us.com","kmoad@insilico.us.com","kmoad@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","support@cravat.us","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@mail.insilico.us.com","support@cravat.us","rkim@insilico.us.com","rkim@insilico.us.com","kmoad@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","mary.a.wood.91@gmail.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","kmoad@insilico.us.com","kmoad@insilico.us.com","rkim@insilico.us.com","support@cravat.us","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com","rkim@insilico.us.com"],["1.3.1","1.0.0","1.0.2","1.3.1","1.0.0","3.6.0","2019.05.13","1.6.1","1.0.0","1.0.0","85.0.12","1.1.0","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.3","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.1.4","1.0.10","1.1.4","1.0.10","1.0.14","1.0.10","1.0.0","2020.02.11.1","91.0.0","91.0.0","1.1.2","2.0.0","1.0.0","151.0.9","1.1.1","1.6.10","1.0.0","6500.2.6","3.5.7","0.4.0","1.0.7","2.3.5","0.0.3","1.3.1","1.0.0","3.5.6","0.0.1","3.6.0","2.1.8","1.0.0","2.2.0","2019.08.16","2.0.7","7.0.4","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.5","1.0.0","1.9.6","1.0.0","1.0.0","2019.08.23","71.1.0","2019.05.20","2019.05.13","1.0.0","3.6.0","1.0.0","1.0.0","1.0.0","4.3.2","3.0.6","1.0.0","1.2.6","2019.08.02","2019.08.19","4.0.11","1.0.0","1.0.0","1.1.2","1.0.0","1.0.0","3.5.7","0.0.8","3.6.0","3.5.8","1.0.0","3.6.0","28.0.1","1.1.4","2019.08.22","2019.05.13","3.1.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","1.0.0","3.5.5","1.0.1","1.1.4","3.1.0","1.0.7","4.0.5","3.1.3","3.1.4","3.1.3","3.1.3","3.1.0","3.1.3","1.0.0","2.0.0","3.5.9","2020.08.10","1.0.0","1.5.2","1.1.5","2.1.0","4.3.0","2019.08.02","1.1.3","1.1.0","1.0.1","1.0.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.8","1.1.0","1.1.0","1.1.0","1.1.0","1.1.3","1.1.0","1.1.0","1.1.0","1.7.0","1.0.1","1.1.0","1.1.0","1.1.0","1.0.0","1.0.3","1.0.3","1.0.2","1.0.0","1.1.0","1.1.0","1.0.8","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.0.3","1.2.0","1.1.0","1.1.0","71.1.0","1.1.0","1.0.2","1.3.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.1","1.1.1","1.0.4","1.0.0","1.1.0","1.1.0","1.1.0","1.0.2","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.1.0","1.3.0","1.3.0","1.1.0","1.0.1","1.1.0","2.0.2","1.1.0","1.1.0","1.0.3","1.1.0"],["23andme-converter","abraom","aggregator","ancestrydna-converter","arrvars","biogrid","brca1_func_assay","cadd_exome","cancer_hotspots","cardioboost","cgc","cgl","chasmplus","chasmplus_ACC","chasmplus_ACC_mski","chasmplus_BLCA","chasmplus_BLCA_mski","chasmplus_BRCA","chasmplus_BRCA_mski","chasmplus_CESC","chasmplus_CESC_mski","chasmplus_CHOL","chasmplus_CHOL_mski","chasmplus_COAD","chasmplus_COAD_mski","chasmplus_DLBC","chasmplus_DLBC_mski","chasmplus_ESCA","chasmplus_ESCA_mski","chasmplus_GBM","chasmplus_GBM_mski","chasmplus_HNSC","chasmplus_HNSC_mski","chasmplus_KICH","chasmplus_KICH_mski","chasmplus_KIRC","chasmplus_KIRC_mski","chasmplus_KIRP","chasmplus_KIRP_mski","chasmplus_LAML","chasmplus_LAML_mski","chasmplus_LGG","chasmplus_LGG_mski","chasmplus_LIHC","chasmplus_LIHC_mski","chasmplus_LUAD","chasmplus_LUAD_mski","chasmplus_LUSC","chasmplus_LUSC_mski","chasmplus_MESO","chasmplus_MESO_mski","chasmplus_OV","chasmplus_OV_mski","chasmplus_PAAD","chasmplus_PAAD_mski","chasmplus_PCPG","chasmplus_PCPG_mski","chasmplus_PRAD","chasmplus_PRAD_mski","chasmplus_READ","chasmplus_READ_mski","chasmplus_SARC","chasmplus_SARC_mski","chasmplus_SKCM","chasmplus_SKCM_mski","chasmplus_STAD","chasmplus_STAD_mski","chasmplus_TGCT","chasmplus_TGCT_mski","chasmplus_THCA","chasmplus_THCA_mski","chasmplus_THYM","chasmplus_THYM_mski","chasmplus_UCEC","chasmplus_UCEC_mski","chasmplus_UCS","chasmplus_UCS_mski","chasmplus_UVM","chasmplus_UVM_mski","chasmplus_group","chasmplus_mski","chasmplus_mski_group","civic","civic_gene","clingen","clinvar","cosmic","cosmic_gene","cravat-converter","csvreporter","cvdkp","dbsnp","dbsnp-converter","denovo","encode_tfbs","esp6500","ess_gene","exac_gene","excelreporter","fathmm","flank_seq","ftdna-converter","genehancer","gerp","geuvadis","ghis","gnomad","gnomad3","gnomad_gene","go","grasp","gtex","gwas_catalog","haploreg_afr","haploreg_amr","haploreg_asn","haploreg_eur","haploreg_group","hg19","hg19wgs","hg38","hg38wgs","hgdp","intact","interpro","javierre_promoters","linsight","litvar","loftool","metalr","metasvm","mirbase","mupit","mutation_assessor","mutpanning","mutpred1","ncbigene","ncrna","ndex","ndex_group","ndex_signor","oldcravat-converter","omim","pharmgkb","phastcons","phdsnpg","phi","phylop","polyphen2","prec","pseudogene","pubmed","repeat","revel","rvis","segway","segway_adrenal","segway_blood","segway_brain","segway_breast","segway_esc","segway_esc_derived","segway_fat","segway_gi_colon","segway_gi_duodenum","segway_gi_intestine","segway_gi_rectum","segway_gi_stomach","segway_group","segway_heart","segway_ipsc","segway_kidney","segway_liver","segway_lung","segway_muscle","segway_muscle_leg","segway_ovary","segway_placenta","segway_skin","segway_stromal_connective","segway_thymus","segway_vascular","sift","siphy","stdoutreporter","tagsampler","target","textreporter","thousandgenomes","thousandgenomes_ad_mixed_american","thousandgenomes_african","thousandgenomes_east_asian","thousandgenomes_european","thousandgenomes_group","thousandgenomes_south_asian","trinity","tsvreporter","uk10k_cohort","uniprot","varmeta","vcf-converter","vcfinfo","vcfreporter","vest","vista_enhancer","wgbase","wgbiogrid","wgbrca1_func_assay","wgcancer_hotspots","wgcgc","wgcgl","wgcglsummary","wgchasmplus","wgcircossummary","wgcivic","wgcivic_gene","wgclingen","wgclinvar","wgcodingvsnoncodingsummary","wgcosmic","wgcosmic_gene","wgdbsnp","wgdenovo","wgesp6500","wgess_gene","wgexac_gene","wgfathmm","wggenehancer","wggerp","wgghis","wggnomad","wggnomad3","wggnomad_gene","wggo","wggosummary","wggrasp","wggtex","wggwas_catalog","wghaploreg_afr","wghaploreg_amr","wghaploreg_asn","wghaploreg_eur","wghaplotypes","wghg19","wghgvs","wgigv","wgintact","wginterpro","wglinsight","wgloftool","wglollipop","wgmupit","wgmutation_assessor","wgmutpred1","wgncbigene","wgncrna","wgndex","wgndexchasmplussummary","wgndexvestsummary","wgnote","wgomim","wgpharmgkb","wgphastcons","wgphdsnpg","wgphi","wgphylop","wgprec","wgpseudogene","wgpubmed","wgrevel","wgrvis","wgsiphy","wgsosamplesummary","wgsosummary","wgtarget","wgthousandgenomes","wgtopgeneschasmplussummary","wgtopgenessummary","wgtopgenesvestsummary","wguk10k_cohort","wgvcfinfo","wgvest"],["Allows user to input files in 23andMe format. ","Exomic allele frequencies of an elderly cohort of Brazilians in the ABraOM database","Aggregator","Allows user to input files in AncestryDNA format. ","Examines variants associated with arrhythmia diseases such as Brugada Syndrome and Lo...","Comprehensive interaction repository","Impact of BRCA1 missense variants on protein function as determined by multiplex homo...","CADD is a tool for scoring the deleteriousness of single nucleotide variants in the h...","A resource for statistically significant mutations in cancer.","Predicts pathogenicity of missense variants for inherited cardiac conditions","Cancer Gene Census annotation","Oncogenes and tumor supressor genes from Vogelstein et al. 2013","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","CHASMplus is a machine learning algorithm that discriminates somatic missense mutatio...","Provides descriptions and linkouts to CIViC","Provides descriptions and linkouts to CIViC","ClinGen is a National Institutes of Health (NIH)-funded resource dedicated to buildin...","ClinVar is an archive of reports of the relationships among human variations and phen...","Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensi...","Catalogue Of Somatic Mutations In Cancer, is the world's largest and most comprehensi...","File converter for standard cravat input","CSV format text file reporter","Produces an effect weight for the variant in which disease it is found. Weights are a...","Comprehensive database of both single base nucleotide subsitutions and short deletion...","Input converter for dbSNP rsid input","Denovo annotation. Denovo is a collection of germline variants in the human genome wh...","Human transcription factor binding sites based on ChIP-seq experiments generated by p...","Exome Sequencing Project 6500","Essential Genes provides a database of genetic variation &amp; mutational burden in 2472 ...","ExAC Functional Gene Constraint &amp; CNV Scores provides probability of LoF tolerance/in...","Microsoft Excel reporter","Functional analysis through hidden markov models.","Reference and alternate bases with flanking sequence","Allows user to input files in FamilyTreeDNA format. ","GeneHancer is a database of genome-wide enhancer-to-gene and promoter-to-gene associa...","Genomic Evolutionary Rate Profiling","Significant snp-gene associations reported by the Geuvadis project","Gene haploinsufficiency scoring","Genome Aggregation Database (gnomAD) is a resource developed by an international coal...","Genome Aggregation Database (gnomAD) is a resource developed by an international coal...","Gene level population statistics from gnomAD","Gene Ontology (GO) project provides a comprehensive computable knowledge regarding th...","Genome-Wide Repository of Associations Between SNPs and Phenotypes","Genotype-tissue expression","GWAS Catalog Annotator","SNPs in LD with the variant in African populations","SNPs in LD with the variant in American populations","SNPs in LD with the variant in Asian populations","SNPs in LD with the variant in European populations","SNPs in LD with the variant","Input coordinates are mapped to hg19 through liftOver.","UCSC hg19 whole genome sequence module","Gene mapper using UCSC Gencode","UCSC hg38 whole genome sequence module","Human Genome Diversity Panel (HGDP) DNA from individuals among 51 different populatio...","Molecular interaction data","Protein sequence analysis &amp; classification","Javierre et al promoter-interacting regions | Promoter capture Hi-C was used to ident...","LINSIGHT","LitVar allows the search and retrieval of variant relevant information from biomedica...","Gene intolerance score based on loss-of-function variants","MetaLR creates an ensemble-based prediction score by using machine learning and logis...","MetaSVM creates an ensemble-based prediction score by using a support vector machine ...","microRNA database of published miRNA sequences and annotation.","Provides links to variant position and annotations in 3D MuPIT structure viewer.","Mutation Assessor is a database providing prediction of the functional impact of amin...","Discovers new tumor genes in aggregated sequencing data.","MutPred is a random forest model for the prediction of pathogenic missense variants a...","Gene description from NIH's NCBI Gene Info","Non-coding RNA from RepeatMasker library","Network Data Exchange","NDEx Annotation Modules","Network Data Exchange SIGNOR Pathways","File converter for old architecture cravat input","Online Mendelian Inheritance in Man. Catalog of human genes and genetic disorders and...","Clinically actionable gene-drug associations and genotype-phenotype relationships","A database of compressed phastCons conservation scores.","A binary classifier for predicting pathogenic variants in coding and non-coding regio...","Providing probabilities for gene-based haploinsufficiency","A database providing conservation scoring by phylogenetic p-values","PolyPhen-2 (Polymorphism Phenotyping v2) is a tool which predicts possible impact of ...","Providing probabilities for LoF alleles","Pseudogenes from UCSC hg38 wgEncodeGencodePseudoGeneV27","PubMed articles related to a particular gene","Repeating elements from RepeatMasker and Simple Repeat tracks of UCSC hg38 database. ...","Rare Exome Variant Ensemble Learner","Residual variation intolerance scoring","Chromatin state activity scores","Chromatin state for Adrenal tissues","Chromatin state for Blood tissues","Chromatin state for Brain tissues","Chromatin state for Breast tissues","Chromatin state for Esc tissues","Chromatin state for Esc Derived tissues","Chromatin state for Fat tissues","Chromatin state for GI Colon tissues","Chromatin state for GI Duodenum tissues","Chromatin state for GI Intestine tissues","Chromatin state for GI Rectum tissues","Chromatin state for GI Stomach tissues","Chromatin state annotations from Segway","Chromatin state for Heart tissues","Chromatin state for Ipsc tissues","Chromatin state for Kidney tissues","Chromatin state for Liver tissues","Chromatin state for Lung tissues","Chromatin state for Muscle tissues","Chromatin state for Muscle Leg tissues","Chromatin state for Ovary tissues","Chromatin state for Placenta tissues","Chromatin state for Skin tissues","Chromatin state for Stromal Connective tissues","Chromatin state for Thymus tissues","Chromatin state for Vascular tissues","SIFT predicts whether an amino acid substitution affects protein function","A database of conservation scores based on 29 mammal genomes","STDOUT reporter","Aggregates tags and samples","Tumor Alterations Relevant for GEnomics-driven Therapy","Text format reporter","Population allele frequencies from 1000 Genomes Project","Population allele frequencies from 1000 Genomes Project","Population allele frequencies from 1000 Genomes Project","Population allele frequencies from 1000 Genomes Project","Population allele frequencies from 1000 Genomes Project","Population allele frequencies from 1000 Genomes Project","Population allele frequencies from 1000 Genomes Project","Trinity assembles transcript sequences from Illumina RNA-Seq data.","Tab-delimited format text file reporter","UK10K cohort provides genetic information from DTR &amp; ALSPAC","Comprehensive resource for protein sequence and functional information.","Aggregates tags and samples","Converter for VCF format input","Writes VCF file information extracted by converter","VCF format text file reporter","VEST annotation","Experimentally validated enhancers from the VISTA Enhancer Browser","Variant annotation webviewer widget","BioGRID webviewer widget","BRCA1 Functional Assay webviewer widget","Cancer Hotspots webviewer widget","Cancer genome census webviewer widget","Cancer gene landscape webviewer widget","Cancer Gene Landscape Summary result viewer widget","CHASMplus webviewer widget","Circos Summary result viewer widget","CIViC webviewer widget","CIViC Gene webviewer widget","Base information webviewer widget","Clinvar webviewer widget","Coding vs noncoding summary webviewer widget","Cosmic webviewer widget","Cosmic gene webviewer widget","dbSNP webviewer widget","denovo-db webviewer widget","ESP6500 webviewer widget","Essential Gene webviewer widget","ExAC Gene widget","FATHMM webviewer widget","GeneHancer webviewer widget","GERP++ web viewer widget","GHIS webviewer widget","gnomAD webviewer widget","gnomAD3 webviewer widget","gnomAD Gene webviewer widget","Gene ontology webviewer widget","Gene Ontology Summary webviewer widget","GRASP webviewer widget","GTEx webviewer widget","GWAS Catalog webviewer widget","Haploreg African information webviewer widget","Haploreg American information webviewer widget","Haploreg Asian information webviewer widget","Haploreg European information webviewer widget","Haplotype information webviewer widget","hg19 info webviewer widget","HGVS webviewer widget","igv.js integration into Open-CRAVAT","IntAct webviewer widget","InterPro webviewer widget","LINSIGHT webviewer widget","LoFtool webviewer widget","Protein diagram webviewer widget","MuPIT webviewer widget","Mutation Assessor webviewer widget","MutPred webviewer widget","NCBI Gene webviewer widget","Non-coding RNA webviewer widget","NDEx webviewer widget","NDEX Enriched Cancer Pathways widget","NDEX Enriched Pathogentic Pathways widget","Note-taking widget","OMIM webviewer widget","PharmGKB webviewer widget","phastCons web viewer widget","PhDSNPg","P(HI) webviewer widget","PhyloP webviewer widget","P(rec) webviewer widget","Pseudogene webviewer widget","PubMed webviewer widget","REVEL webviewer widget","RVIS webviewer widget","SiPhy webviewer widget","Sequence Ontology Sample Summary webviewer widget","Sequence Ontology Summary result viewer widget","Target webviewer widget","1000 genomes webviewer widget","Top Genes CHSAMplus Summary webviewer widget","Top Genes Summary webviewer widget","Top Genes VEST Summary webviewer widget","UK10K Cohort webviewer widget","VCF info webviewer widget","VEST webviewer widget"],["2019-05-22T19:12:35.744619+0000","2020-08-21T18:27:35.853157+0000","2018-11-20T22:16:49.000000","2019-05-22T19:12:52.009274+0000","2020-08-17T15:23:15.712640+0000","2019-01-30T18:34:54.833895+0000","2019-03-23T20:11:20.673046+0000","2020-05-18T16:24:17.165379+0000","2020-08-17T15:24:05.845210+0000","2020-08-24T18:16:30.890234+0000","2019-03-23T17:50:47.555923+0000","2019-01-30T18:35:22.551632+0000","2019-01-30T18:35:31.269709+0000","2019-05-31T03:21:24.414464+0000","2020-03-06T00:14:16.088529+0000","2019-05-31T03:21:42.309414+0000","2020-03-06T00:14:54.523773+0000","2019-05-31T03:21:59.511102+0000","2020-03-06T00:15:32.665748+0000","2019-05-31T03:22:16.689409+0000","2020-03-06T00:16:10.833024+0000","2019-05-31T03:22:33.864305+0000","2020-03-06T00:16:49.433338+0000","2019-05-31T03:22:51.319119+0000","2020-03-06T00:17:27.396662+0000","2019-05-31T03:23:09.674093+0000","2020-03-06T00:18:05.760902+0000","2019-05-31T03:23:26.881833+0000","2020-03-06T00:18:44.013277+0000","2019-05-31T03:23:44.080965+0000","2020-03-06T00:19:22.540118+0000","2019-05-31T03:24:01.748559+0000","2020-03-06T00:20:01.100507+0000","2019-05-31T03:24:18.962661+0000","2020-03-06T00:20:39.613788+0000","2019-05-31T03:28:30.820328+0000","2020-03-06T00:21:17.971297+0000","2019-05-31T03:24:36.235317+0000","2020-03-06T00:21:56.269875+0000","2019-05-31T03:24:53.891396+0000","2020-03-06T00:22:34.425382+0000","2019-05-31T03:25:11.448773+0000","2020-03-06T00:23:12.593235+0000","2019-05-31T03:28:48.594286+0000","2020-03-06T00:23:50.797367+0000","2019-05-31T03:29:05.627192+0000","2020-03-06T00:24:28.646196+0000","2019-05-31T03:29:22.870408+0000","2020-03-06T00:25:06.797603+0000","2019-05-31T03:29:39.953342+0000","2020-03-06T00:25:45.470985+0000","2019-05-31T03:29:57.575185+0000","2020-03-06T00:26:23.481498+0000","2019-05-31T03:30:14.724908+0000","2020-03-06T00:27:01.366449+0000","2019-05-31T03:30:48.951257+0000","2020-03-06T00:27:39.688392+0000","2019-05-31T03:31:06.661266+0000","2020-03-06T00:28:18.248150+0000","2019-05-31T03:31:23.976942+0000","2020-03-06T00:28:56.783490+0000","2019-05-31T03:31:41.144509+0000","2020-03-06T00:35:19.350673+0000","2019-05-31T03:31:58.714812+0000","2020-03-06T00:29:35.272941+0000","2019-05-31T03:32:16.266425+0000","2020-03-06T00:30:13.566463+0000","2019-05-31T03:32:33.420940+0000","2020-03-06T00:30:52.069041+0000","2019-05-31T03:32:50.659466+0000","2020-03-06T00:31:30.523276+0000","2019-05-31T03:33:07.928708+0000","2020-03-06T00:32:08.612628+0000","2019-05-31T03:33:25.575306+0000","2020-03-06T00:32:46.640665+0000","2019-05-31T03:33:42.780251+0000","2020-03-06T00:33:24.727108+0000","2019-05-31T03:34:00.072936+0000","2020-03-06T00:34:03.079726+0000","2020-08-21T18:34:23.825694+0000","2020-03-11T22:09:48.703509+0000","2020-03-04T13:02:19.979999+0000","2019-01-30T18:35:41.089779+0000","2019-01-30T18:35:50.003914+0000","2020-08-21T18:36:56.505747+0000","2020-03-04T23:05:21.636868+0000","2020-08-21T18:38:38.757697+0000","2020-08-21T18:39:30.060400+0000","2018-12-10T16:51:10.000000","2019-06-12T17:11:21.008048+0000","2020-08-21T18:41:12.031618+0000","2019-01-30T18:37:40.376164+0000","2019-09-17T16:37:16.536626+0000","2019-01-30T18:37:48.877593+0000","2020-08-21T18:44:35.954192+0000","2019-01-30T01:56:13.545867+0000","2019-01-30T18:39:07.249427+0000","2019-01-30T18:39:16.427766+0000","2018-08-14T15:05:24.000000","2019-01-30T18:39:44.613445+0000","2019-03-07T18:46:08.026166+0000","2020-04-01T10:30:57.549546+0000","2020-09-01T19:35:29.503611+0000","2019-01-30T18:39:54.218255+0000","2020-08-21T18:51:24.668465+0000","2019-01-30T18:40:02.446704+0000","2019-01-30T18:40:12.357216+0000","2020-08-21T18:53:57.553853+0000","2019-01-30T18:40:20.370464+0000","2019-01-29T19:34:12.000000","2019-01-30T18:41:07.241162+0000","2019-01-30T19:10:29.903408+0000","2020-08-21T18:58:13.441074+0000","2019-10-31T02:43:51.032564+0000","2019-10-31T02:44:22.685664+0000","2019-10-31T02:44:52.930937+0000","2019-10-31T02:45:23.508973+0000","2020-08-21T18:59:04.386705+0000","2019-11-26T19:57:36.190745+0000","2020-05-18T19:08:14.663706+0000","2018-11-16T17:09:25.000000","2020-05-18T19:08:58.046364+0000","2020-08-21T19:00:46.325230+0000","2019-01-30T18:41:50.829659+0000","2019-01-30T18:41:58.811979+0000","2019-02-11T20:29:36.950176+0000","2018-11-01T20:59:34.000000","2020-09-04T21:02:35.734948+0000","2019-01-30T18:42:16.687716+0000","2020-09-01T19:36:22.058183+0000","2020-09-01T19:40:55.226472+0000","2020-08-31T15:01:17.047131+0000","2019-01-30T18:42:26.597314+0000","2019-01-30T18:42:36.301616+0000","2020-08-21T19:06:43.062461+0000","2019-01-30T19:20:23.202995+0000","2019-03-25T17:51:20.304381+0000","2019-01-30T20:42:51.131258+0000","2019-01-30T01:56:40.452873+0000","2020-08-18T00:13:32.961417+0000","2020-08-18T00:46:34.699489+0000","2019-01-07T22:43:03.000000","2020-08-21T19:10:57.862193+0000","2020-08-21T19:11:49.283863+0000","2019-01-30T18:43:56.366250+0000","2019-01-30T18:44:04.627754+0000","2019-01-30T18:44:14.269915+0000","2019-03-23T18:13:12.105921+0000","2020-08-21T19:16:04.627843+0000","2019-01-28T21:42:44.000000","2019-01-30T18:44:48.081851+0000","2018-09-27T17:48:28.000000","2019-01-30T18:45:06.202135+0000","2020-08-21T19:20:19.348073+0000","2019-01-30T18:45:23.421954+0000","2019-09-25T00:17:24.388739+0000","2019-09-25T00:18:01.170904+0000","2019-09-25T00:18:29.289790+0000","2019-09-25T00:18:57.705488+0000","2019-09-25T00:19:26.287984+0000","2019-09-25T00:19:54.441801+0000","2019-09-25T00:20:22.748135+0000","2019-09-25T00:20:51.333749+0000","2019-09-25T00:21:19.647448+0000","2019-09-25T00:21:48.334100+0000","2019-09-25T00:22:16.983203+0000","2019-09-25T00:22:45.501876+0000","2019-09-25T00:23:14.286254+0000","2020-08-21T19:22:01.161351+0000","2019-09-25T00:24:12.187382+0000","2019-09-25T00:24:41.308084+0000","2019-09-25T00:25:10.490444+0000","2019-09-25T00:25:39.487000+0000","2019-09-25T00:26:08.539041+0000","2019-09-25T00:26:37.689786+0000","2019-09-25T00:27:06.839385+0000","2019-09-25T00:27:36.188141+0000","2019-09-25T00:28:05.718459+0000","2019-09-25T00:28:35.234702+0000","2019-09-25T00:29:04.808046+0000","2019-09-25T00:29:34.371132+0000","2019-09-25T00:30:03.689008+0000","2020-08-21T19:22:52.131743+0000","2019-01-30T18:45:33.527748+0000","2020-07-14T13:33:05.362044+0000","2018-07-24T19:44:47.000000","2019-01-30T18:45:59.292148+0000","2018-08-14T15:06:06.000000","2019-01-30T18:46:33.377326+0000","2019-06-07T18:59:59.557670+0000","2019-06-07T19:00:17.976738+0000","2019-06-07T19:00:35.927973+0000","2019-06-07T19:00:53.753998+0000","2020-08-21T19:25:24.792063+0000","2019-06-07T19:15:43.009947+0000","2020-08-21T19:26:15.735084+0000","2019-06-12T17:11:00.368964+0000","2019-01-30T18:46:41.583710+0000","2019-12-18T16:06:24.880670+0000","2020-03-17T19:21:41.695399+0000","2018-12-21T16:49:41.000000","2018-05-18T14:15:57.000000","2020-01-30T19:14:31.701795+0000","2019-01-30T18:47:42.647635+0000","2019-02-11T20:10:23.260495+0000","2018-11-01T00:15:58.000000","2019-01-08T21:36:07.000000","2019-05-06T20:14:26.847780+0000","2020-08-06T19:22:32.763140+0000","2019-05-06T20:14:42.564704+0000","2019-12-18T18:42:29.599815+0000","2018-10-26T20:43:31.000000","2018-10-26T20:46:00.000000","2018-09-26T01:39:06.000000","2018-10-26T20:46:25.000000","2018-10-26T20:46:48.000000","2019-10-16T03:24:22.550948+0000","2019-01-08T21:10:10.000000","2018-09-26T01:39:16.000000","2019-05-06T20:16:28.419376+0000","2019-01-09T15:45:32.000000","2018-10-26T20:48:34.000000","2018-08-16T21:54:54.000000","2019-05-06T20:17:28.803767+0000","2018-12-05T18:37:27.000000","2019-12-18T18:47:15.835464+0000","2018-11-26T19:32:50.000000","2020-08-28T20:01:04.000676+0000","2018-12-11T20:29:41.000000","2018-11-29T17:04:37.000000","2018-10-26T20:49:59.000000","2020-04-17T20:02:02.414300+0000","2019-05-06T20:19:14.905227+0000","2018-05-18T14:23:01.000000","2018-10-26T20:50:29.000000","2018-10-26T20:50:48.000000","2019-03-11T12:00:46.498740+0000","2019-08-26T15:34:53.999669+0000","2019-10-25T21:28:47.289373+0000","2019-10-25T21:29:18.482192+0000","2019-10-25T21:29:48.559944+0000","2019-10-25T21:30:18.994356+0000","2019-05-06T20:20:30.324476+0000","2018-11-13T18:48:37.000000","2018-10-26T20:51:08.000000","2018-11-13T21:20:38.000000","2019-01-08T21:38:13.000000","2019-01-10T14:43:23.000000","2018-10-26T20:51:29.000000","2018-12-11T20:29:31.000000","2018-11-14T22:12:07.000000","2018-10-26T20:51:53.000000","2018-11-26T19:33:16.000000","2018-11-28T16:34:00.000000","2019-05-06T20:23:02.251057+0000","2018-10-26T20:53:07.000000","2018-10-26T20:53:26.000000","2018-09-26T01:40:59.000000","2018-09-26T01:41:16.000000","2019-01-12T16:58:39.000000","2020-07-21T22:47:32.902499+0000","2019-09-30T19:48:42.874848+0000","2018-11-26T19:53:49.000000","2018-10-26T20:53:55.000000","2019-01-08T21:39:02.000000","2019-01-03T20:32:39.000000","2019-01-08T21:39:35.000000","2018-10-26T20:54:52.000000","2019-05-06T20:25:18.796147+0000","2018-10-26T20:55:49.000000","2019-01-09T16:07:12.000000","2019-05-06T20:26:19.430462+0000","2018-09-26T01:41:46.000000","2018-09-26T01:41:51.000000","2019-05-06T20:27:05.179368+0000","2019-05-06T20:27:21.302904+0000","2018-10-16T18:07:44.000000","2018-09-26T01:42:15.000000","2018-10-16T18:07:59.000000","2019-05-06T20:27:52.817920+0000","2018-11-13T18:45:14.000000","2018-10-29T20:25:54.000000"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>title<\/th>\n      <th>type<\/th>\n      <th>developer<\/th>\n      <th>latest_version<\/th>\n      <th>name<\/th>\n      <th>description<\/th>\n      <th>published<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"lengthMenu":[5,10,25,100],"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="single-variant-queries" class="section level2">
<h2 class="hasAnchor">
<a href="#single-variant-queries" class="anchor"></a>Single-variant queries</h2>
<p>The <code>queryOC</code> function uses the OpenCRAVAT REST API to acquire information on a single position in GRCh38 coordinates. This function requires that a registered username and password be supplied. Visit <code>run.opencravat.org</code> to set a username and password.</p>
<p>An example API response is available in <code>var_in_tx</code> in the <code>oc2bioc</code> package:</p>
<div class="sourceCode" id="cb2"><pre class="downlit">
<span class="kw">var_in_tx</span>
<span class="co">#&gt; Response [https://run.opencravat.org/submit/annotate?chrom=chr7&amp;pos=140753336&amp;ref_base=A&amp;alt_base=T&amp;annotators=chasmplus_BRCA,pubmed,segway_breast]</span>
<span class="co">#&gt;   Date: 2020-09-07 10:33</span>
<span class="co">#&gt;   Status: 200</span>
<span class="co">#&gt;   Content-Type: application/json; charset=utf-8</span>
<span class="co">#&gt;   Size: 1.31 kB</span>
</pre></div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Vincent Carey.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.0.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
